test/api-digester/stability-stdlib-abi-with-asserts.test

```diff
@@ -38,7 +38,7 @@
 // Thank you for your help ensuring the stdlib remains compatible with its past!
 //                                            -- Your friendly stdlib engineers
 
-// REQUIRES: swift_stdlib_asserts
+// REQUIRES: swift_stdlib_asserts, array_cow_checks
```

Comment:
Does this mean that the ABI checks won't run by default when we run the tests locally?

Reply:
yes. Is there a way to make these symbols "optional"?